### PR TITLE
Push to Production

### PR DESCRIPTION
**Pull request by aallan**

### CODE REVIEW

File: documentation/asciidoc/microcontrollers/microcontroller_docs.adoc [hunk 1]
```diff
@@ -24,4 +24,9 @@ https://datasheets.raspberrypi.org/pico/raspberry-pi-pico-c-sdk.pdf[Raspberry Pi
 
 https://datasheets.raspberrypi.org/pico/raspberry-pi-pico-python-sdk.pdf[Raspberry Pi Pico Python SDK]:: A MicroPython environment for RP2040 microcontrollers
 
-The API level Doxygen documentation for the Raspberry Pi Pico C/C++ SDK is also available https://rptl.io/pico-doxygen[as a micro-site].
\ No newline at end of file
+The API level Doxygen documentation for the Raspberry Pi Pico C/C++ SDK is also available https://rptl.io/pico-doxygen[as a micro-site].
```
Comment (lurch) [inline documentation/asciidoc/microcontrollers/microcontroller_docs.adoc:27]: Should API-level be hyphenated?

File: documentation/asciidoc/microcontrollers/microcontroller_docs.adoc [hunk 2]
```diff
@@ -24,4 +24,9 @@ https://datasheets.raspberrypi.org/pico/raspberry-pi-pico-c-sdk.pdf[Raspberry Pi
...
+
+[NOTE]
+======
+If you are building applications with the C/C++ SDK and targetting boards other than the Raspberry Pi Pico, you will need to pass `-DPICO_BOARD=boardname` to CMake. Here `boardname` is the name of your board, e.g. for the Adafruit Feather RP2040 you should pass `-DPICO_BOARD=adafruit_feather_rp2040_rp2040`. See the https://github.com/raspberrypi/pico-sdk/tree/master/src/boards[`boards/` directory] in the Pico SDK, and the https://www.raspberrypi.org/forums/viewtopic.php?f=147&t=304393[forums], for more information.
```
Comment (lurch) [inline documentation/asciidoc/microcontrollers/microcontroller_docs.adoc:31]: Are you sure `adafruit_feather_rp2040_rp2040` is right? :wink:

Comment (aallan) [reply documentation/asciidoc/microcontrollers/microcontroller_docs.adoc:31]: Fixed in 5333297.

File: documentation/asciidoc/microcontrollers/microcontroller_docs.adoc (hunk 1 above)
Comment (aallan) [reply documentation/asciidoc/microcontrollers/microcontroller_docs.adoc:27]: Not hyphenated on the RP2040 Landing Page where this text came from and that's been through multiple copyedits by the comms team, so I'm guessing not. I'll ask Helen and Liz though.

**Pull request merged**